python/cuml/dask/manifold/base.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common.base import BaseEstimator
+
+
+class BaseManifold(BaseEstimator):
+
+    def __init__(self, client=None, **kwargs):
+        """
+        Constructor for distributed decomposition model
+        """
+        super(BaseManifold, self).__init__(client, **kwargs)
+
+        # define attributes to make sure they
+        # are available even on untrained object
+        self.local_model = None
```

Comment:
I believe this should be done in `cuml.dask.Base`, it it's not already. If this is the only reason for having a shared base class, I'm thinking maybe we should just remove it. What do you think?

Reply:
Agree